org.gnome.Eolie.json

```diff
@@ -0,0 +1,314 @@
+{
+  "app-id": "org.gnome.Eolie",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "eolie",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:rw",
+    "--talk-name=org.freedesktop.secrets",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+  ],
+  "modules": [
+    {
+      "name": "gst-libav",
```

Comment:
Ok, I think I understand, libav(ffmpeg) provides codecs provided by gstreamer.
Last question, do I really need to build ffmpeg separated like in GnomeTwitch, seems gst-libav can use directly build options, no?